### PR TITLE
Penv 3 configs

### PR DESCRIPTION
**Pull request by Vany**
update for penv-3 requirements

### CODE REVIEW

File: run-test.py [hunk 1]
```diff
@@ -820,6 +820,7 @@ def deploy_postgresql(pod, service_name):
 def deploy_observer_deps():
     deploy_postgresql(OBSERVER, 'observer')
     info("starting Nginx @ pod "+str(OBSERVER))
+    ssh(OBSERVER, """sudo bash -c \\"apt update\\" """)
```
Comment (twwlf) [inline run-test.py:823]: I suggest installing postgres & nginx in base image instead of storing just cache and updating index on this step

Comment (Vany) [reply run-test.py:823]: yes, ask @afiskon for this. but is not a subject of this changes.

Comment (twwlf) [reply run-test.py:823]: Well, if it isn't a subject, why this code present?

File: run-test.py [hunk 2]
```diff
@@ -889,29 +920,13 @@ def deploy_insolar(skip_benchmark=False, use_postgresql=False):
...
-            scp_to(pod, "/tmp/insolar-jepsen-configs/insolar_" +
-                   str(pod)+".yaml", pod_path)
-        scp_to(pod, "/tmp/insolar-jepsen-configs/pulsewatcher.yaml",
-               INSPATH+"/pulsewatcher.yaml")
-        scp_to(pod, "/tmp/insolar-jepsen-configs/pulse_exporter.proto",
-               INSPATH+"/pulse_exporter.proto")
+        queue.put(pod)
+        t = Thread(target=deploy_insolar_for_each_pod, args=(queue, pod_ips, use_postgresql))
+        t.start()
+        #deploy_insolar_for_each_pod(queue, pod_ips, use_postgresql)
```
Comment (megge-dream) [inline run-test.py:928]: if you decide to use threads, delete this commented line

Comment (Vany) [reply run-test.py]: you are right. done

File: run-test.py (hunk 1 above)
Comment (Vany) [reply run-test.py]: this changes was made to allow this code to run in strange network environment. seems it works without it right now

File: run-test.py [hunk 3]
```diff
@@ -851,13 +851,13 @@ def deploy_observer(path, keep_database=False):
             "/../observer && GO111MODULE=on make migrate")
     # run observer
     ssh(OBSERVER, """tmux new-session -d -s observer \\"cd """+INSPATH +
-        """/../observer && ./bin/observer 2>&1 | tee -a observer.log; bash\\" """)
+        """/../observer && ./bin/observer ./artifacts/observer.yaml 2>&1 | tee -a observer.log; bash\\" """)
```
Comment (twwlf) [inline run-test.py:854]: we are setting path to config without `--config` flag, aren't we?

Comment (megge-dream) [reply run-test.py:854]: also, path should be `.artifacts/observer.yam`l

File: run-test.py [hunk 4]
```diff
@@ -881,6 +881,36 @@ def gen_certs():
         scp_to(pod, "/tmp/insolar-jepsen-configs/reusekeys/discovery/*",
                INSPATH+"/scripts/insolard/reusekeys/discovery/")
 
+from queue import Queue
```
Comment (twwlf) [inline run-test.py:884]: sudden queuing bothers me, seems a bit risky, race-y. Also, this is not related to the original task, as far as i know.

Comment (Vany) [reply run-test.py]: ok, let's simplify your life
